engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -287,9 +287,10 @@ private void addShadowMapNodes(RenderGraph renderGraph) {
         renderGraph.addNode(shadowMapNode, "shadowMapNode");
     }
 
-    private void addReflectionNodes(RenderGraph renderGraph) {
+    private void addReflectionNodes() {
         FBOConfig reflectedBufferConfig = new FBOConfig(BackdropReflectionNode.REFLECTED_FBO_URI, HALF_SCALE, FBO.Type.DEFAULT).useDepthBuffer();
-        BufferClearingNode reflectedBufferClearingNode = new BufferClearingNode(reflectedBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+        BufferClearingNode reflectedBufferClearingNode = new BufferClearingNode(reflectedBufferConfig, displayResolutionDependentFBOs,
```

Comment:
Is this a line length problem? If so, I'd start the second line with the "new" keyword, so that each side of the assignment is on one line. Other lines below have similar issues.

Reply:
Yes, it goes past the 175 character limit and I have just adjusted the line breaks. Also I have play tested it to confirm that it is indeed functional.